INFO     root:conftest.py:184 === Starting auth_token Test ===
INFO     root:conftest.py:185 API Response: 200 | {"kind":"identitytoolkit#VerifyCustomTokenResponse","idToken":"eyJhbGciOiJSUzI1NiIsImtpZCI6IjAzMmNjMWNiMjg5ZGQ0NjI2YTQzNWQ3Mjk4OWFlNDMyMTJkZWZlNzgiLCJ0eXAiOiJKV1QifQ.eyJhZ3JlZW1lbnRzU2lnbmVkIjpmYWxzZSwiZW1haWxWZXJpZmllZCI6dHJ1ZSwibWVtYmVyQmlydGhEYXRlIjoiMDMvMDgvMTk5MSAwMDowMDowMCIsIm1lbWJlcklkIjozODI5LCJtZW1iZXJGaXJzdE5hbWUiOiJRQSIsIm1lbWJlckxhc3ROYW1lIjoiTmVyZCIsIm1lbWJlckRpc3BsYXlOYW1lIjoiTmVyZENvcmUtUUEiLCJtZW1iZXJOYW1lIjoiTmVyZCBDb3JlIiwibWVtYmVyTnVtYmVyIjoiMDAzODI5IiwibWVtYmVyUGhvbmUiOiIrMTMxMDU1NTU1NTUiLCJtZW1iZXJUaW1lWm9uZSI6IiswNzowMCIsInBlcm1pc3Npb25zIjpbXSwicm9sZXMiOltdLCJ1c2VySWQiOiI2NDJlZWZhNjc1ZGU3MDdjNDRhZmM4NDAiLCJvcHRJblNoYXJlTmFtZSI6ZmFsc2UsImF1dGhlbnRpY2F0b3JFbmFibGVkIjpmYWxzZSwiaXNzIjoiaHR0cHM6Ly9zZWN1cmV0b2tlbi5nb29nbGUuY29tL2NvcmUtZGV2LTliNTRiIiwiYXVkIjoiY29yZS1kZXYtOWI1NGIiLCJhdXRoX3RpbWUiOjE3MDMyMjg3MDQsInVzZXJfaWQiOiJvQ0RSbHNNOVMzWjdiejBXNzMyYlBUcWlGUUgzIiwic3ViIjoib0NEUmxzTTlTM1o3YnowVzczMmJQVHFpRlFIMyIsImlhdCI6MTcwMzIyODcwNCwiZXhwIjoxNzAzMjMyMzA0LCJlbWFpbCI6Im5lcmRjb3JlX3FhdGVhbSsxQG5lcmR1bml0ZWQuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnsiZW1haWwiOlsibmVyZGNvcmVfcWF0ZWFtKzFAbmVyZHVuaXRlZC5jb20iXX0sInNpZ25faW5fcHJvdmlkZXIiOiJjdXN0b20ifX0.VMEtI_4kNkfUBCeG3jtkykG1qfvXDw_1AofdyFYsjClZmLJg-bckR5P73xJLvOEUMErW2q5G1wkbCBkE2gHBc9xNrj1r3UIpiINdaTEg7FSsjK-8jmw-YQHB0Ij0-AkUC7_2pnWzrYeAoSYrrhmqSJp2U1KTmdgFH63y3U8LxR5uGz9iTMFqaulrnhklF986K-seyeHLojtObQ6AZHq7t2IxxarPBf969CQCX_W6KXF24pE687mXHfVcDjiG0yJSMuFUEn4wBohXn95X4WAH8yF11OQ4ziGNDAtZSkOcLrZSY3IbrgCf60AP9Scwe-YTauwCbhwdmsxBo-Kmtty62g","refreshToken":"AMf-vBz2ehd4XLcwOlehSsNWqC8jU8sh0Bhn9Kj4fPOycvckwjFqpt6GmleUa8PVOReu84dWLtaeg8MW0JfNogW-eBcPk2n1JkM6E7eD5Azzof4SmXLwa2rSyZ-lLP7ql3bwIcQFczYsZ_jqUqsdlKQKpkrGTdyh3_BktVlkJyKQqIT6LTLZ4yXh4xqfymrXXZjNjT-GFQa1WcpdJ-6x7IPgjGY7h2ZefnB_gqtaZi7MfprnMiC57_shdvgtlejD3KbbivzeGTp3rl_W18E1qk7qb4cOC2xIYWRFgRkEEqqPG4lWv_ATvcI9q_NDPcR3OCz9eVTPFlegJmIBQSVoS1y_lKTx0zxZI9n_fQG5yM4eZJ47Ik9EPsah6JQidtumnRWj8MtYQHqS0-roO_bb9ap-VQzWGmTWxrgl44wXi4t--Ei1a5Rj1xim6UUNwLLJye8JoXmRyt21q8zNGzumQAt7JTEZjXB6LGyvUDCRedOaro0crMspxec5N71JR4u4-ITY8ggfC20RB30S6gaEctbaRXgOr3N4mW1UeM2EQ78KjdjRZ4c-L8OO_z-dQ7gG1xzobgxImOeQS-BfBCzXWEXmSa8-EKIfTg7GIvhluNyirIANrDGjUOi2YpalRSlO4HV6XeyjhjQ2Zi0Cyh4O63b2Xtw7RNLNjjSp3LJZ4ct2GCQnlbE-roT8t9rMxjmBkOe0ecbYFldkDpiyrreGtI6jKQX-XJlkvA","expiresIn":"3600","isNewUser":false}
INFO     root:conftest.py:195 Test 'auth_token' SUCCESS.
INFO     root:conftest.py:196 === Ending auth_token Test ===
INFO     root:conftest.py:184 === Starting test_get_a_role Test ===
INFO     root:conftest.py:185 API Response: 401 | 
INFO     root:conftest.py:191 Test 'test_get_a_role' FAILED.
ERROR    root:conftest.py:192 API response status code: 401
assert 401 in [200, 201, 202, 203, 204, 205]
 +  where 401 = <Response [401]>.status_code
Traceback (most recent call last):
  File "/home/runner/work/qa-nerd-core-backend/qa-nerd-core-backend/conftest.py", line 189, in api_response_conditions
    assert response.status_code in allowed_status_codes, f"API response status code: {response.status_code}"
AssertionError: API response status code: 401
assert 401 in [200, 201, 202, 203, 204, 205]
 +  where 401 = <Response [401]>.status_code